include/filter/filter.hh

```diff
@@ -1,4 +1,35 @@
 #ifndef FILTER_FILTER_HH
 #define FILTER_FILTER_HH
 
-#endif // FILTER_FILTER_HH
+#include "../audio/audio.hh"
+#include "../pipe.hh"
+#include <Iir.h>
+#include <climits>
+#include <condition_variable>
+#include <cstdio>
+#include <memory>
+#include <thread>
```

Comment:
are climits, cstdio, and memory still required?

Reply:
climits is used for default filter values, but could remove the other two.